app/services/data/get-open-reductions-for-court-reporters.js

```diff
@@ -0,0 +1,14 @@
+const knex = require('../../../knex').appSchema
+
+module.exports = function (courtReportStagingIdStart, courtReportStagingIdEnd, workloadReportId) {
+  return knex('reductions')
+    .join('court_reports', 'court_reports.workload_owner_id', 'reductions.workload_owner_id')
+    .select(
+      'reductions.id AS id',
+      'reductions.effective_from AS effectiveFrom',
+      'reductions.effective_to AS effectiveTo',
+      'reductions.status AS status'
+    )
+    .whereRaw('court_reports.staging_id BETWEEN ? AND ? AND (status IS NULL OR status IN (\'ACTIVE\',\'SCHEDULED\')) ' +
```

Comment:
Could use the reduction status constant here instead of 'ACTIVE' etc

Reply:
Done